Sources/Core/Components/PlayerContext.swift

```diff
@@ -32,6 +32,7 @@ protocol PlayerContextDelegate: class {
     func playerContext(didCurrentTimeChange currentTime: Double)
     func playerContext(didItemDurationChange itemDuration: Double?)
     func playerContext(debugMessage: String?)
+    func playerContext(errorInfo: PlayerErrorInfo)
```

Comment:
1. logs can return multiple events so it should be [PlayerErrorInfo]
2. argument name errorInfo is not enough specific. it concern logs of item so may be: itemErrorLogs

Reply:
For me debugMessage and errors are two different things. The client will react differently. But if the change is a must for you I'll adapt the changes